server/test/unit/server/test_logs.py

```diff
@@ -765,8 +763,7 @@ def fake_getLogger(name=None):
         # We should have used the user's setting
         root_logger.setLevel.assert_called_once_with(logging.ERROR)
 
-    @mock.patch('pulp.server.logs.config.config.get',
-                side_effect=ConfigParser.NoOptionError('server', 'log_level'))
+    @mock.patch('pulp.server.logs.config.config.get', return_value='INFO')
```

Comment:
The way that we do default checking with our unit tests is confusing to me. We've had a variety of issues with our unit tests in the past. This test was failing for me and I didn't understand it since we provide a default I never expect a `NoOptionError` to be raised. What I really want to do is to assert that the default value of `get('server', 'log_level')` is INFO.